README.md

```diff
@@ -1 +1,1178 @@
+reporter.api.version="v1"
+
+
+reporter.compression.enabled=false
+```
+
+- Then open the terminal and navigate to `ballerina-honeycomb/guide` and run Ballerina project initializing toolkit.
+
+``
+   $ ballerina init
+``
+- Also you need to clone and build the ballerina-zipkin-extension in the following repository [https://github.com/ballerina-platform/ballerina-observability/tree/master/tracing-extensions/modules.](https://github.com/ballerina-platform/ballerina-observability/tree/master/tracing-extensions/modules) 
+
+- After building  move to `ballerina-zipkin-extension>/target/distribution/` and copy all the jar files to your `bre/lib` folder in your ballerina distribution.
+### Development of student and marks service with Honeycomb
+
+Now let us look into the implementation of the student management with observability.
+
+##### student_management_service.bal
+
+``` ballerina
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/io;
+import ballerina/log;
+import ballerina/mysql;
+import ballerina/observe;
+import ballerina/runtime;
+
+// Type Student is created to store details of a student.
+type Student record {
+    int id;
+    int age;
+    string name;
+    int mobNo;
+    string address;
+};
+
+// End point for marks details client.
+endpoint http:Client marksServiceEP {
+    url: " http://localhost:9191"
+};
+
+// Endpoint for mysql client.
+public endpoint mysql:Client databaseEP {
+    host: "localhost",
+    port: 3306,
+    name: "testdb",
+    username: "root",
+    password: "",
+    poolOptions: { maximumPoolSize: 5 },
+    dbOptions: { useSSL: false }
+};
+
+// This service listener.
+endpoint http:Listener studentServiceListener {
+    port: 9292
+};
+
+// Student data service.
+@http:ServiceConfig {
+    basePath: "/records"
+}
+service<http:Service> StudentData bind studentServiceListener {
+
+    int errors = 0;
+    int requestCounts = 0;
+
+    @http:ResourceConfig {
+        methods: ["POST"],
+        path: "/addStudent"
+    }
+    // Add Students resource used to add student records to the system.
+    addStudents(endpoint httpConnection, http:Request request) {
+        // Initialize an empty http response message.
+        requestCounts++;
+        http:Response response;
+
+        // Accepting the Json payload sent from a request.
```

Comment:
Change "Json" to "JSON". Apply elsewhere if applicable.

Reply:
Noted I will make the above changes.